File: Java/백준/Bronze/29319. Начало/Начало.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int max = 0;
        int sum = 0;
        for (int i=0;i<a;i++){
            int q = in.nextInt();
            max = max>q?max:q;
            sum+=q;
        }
        System.out.println(sum-max);
    }
}

File: C#/백준/Bronze/29319. Начало/Начало.cs
using System;
using System.Linq;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            Console.WriteLine(aa.Sum()-aa.Max());
        }
    }
}